src/test/java/com/intuit/fuzzymatcher/component/MatchServiceTest.java

```diff
@@ -567,6 +567,44 @@ public void itShouldApplyMatchWithDate() {
         Assert.assertEquals(2, result.size());
     }
 
+    @Test
+    public void itShouldApplyMatchWithAge() {
+        List<Object> numbers = Arrays.asList(1, 2, 9, 10, 11, 45, 49, 50, 52, 55, 90, 95, 100, 107, 115);
+        List<Document> documentList1 = getTestDocuments(numbers, AGE, null);
+        Map<Document, List<Match<Document>>> result1 = matchService.applyMatch(documentList1);
+        documentList1.get(0).getElements().iterator().next().getValue();
+        Map<Object, List<Object>> collect = result1.entrySet().stream().
+                collect(Collectors.toMap(e -> e.getKey().getElements().iterator().next().getValue(),
+                        e -> e.getValue().stream().map(m -> m.getMatchedWith().getElements().iterator().next().getValue())
+                                .collect(Collectors.toList())));
+
+        for (Map.Entry<Object, List<Object>> entry : collect.entrySet()) {
+            int doc = (int) entry.getKey();
+            for (Object matchWith: entry.getValue()) {
+                int match = (int) matchWith;
+                double diff = Math.abs(doc - match);
+                Assert.assertTrue(diff >= 1 && diff <=10);
+            }
+        }
+
+        List<Document> documentList2 = getTestDocuments(numbers, NUMBER, null);
```

Comment:
I understand this is a backward compatibility test to ensure NUMBER functionality is not impacted,   can we break this into a separate test for Number ?

Reply:
Added a stricter check on existing unit test.